src/treemodel/treewalker.js

```diff
@@ -44,6 +44,18 @@ export default class TreeWalker {
 			throw new CKEditorError( 'tree-walker-no-start-position: Neither boundaries nor starting position have been defined.' );
 		}
 
+		if ( options && options.direction && [ 'FORWARD', 'BACKWARD' ].indexOf( options.direction ) < 0 ) {
+			/**
+			 * Unknown direction.
```

Comment:
The API doc for an error is supposed to be more verbose than the in-code message. But if the message in the code is enough, there's no point to add an API doc for an error, so you can remove this.